### ISSUE TITLE
How to proxy a repo using docker image

### ISSUE THREAD

**Issue by colinwebber** [dzikoysk/reposilite#111]
I would love to introduce your artifact manager into my company's workflow but am really struggling with the lack of documentation. Currently I need to proxy some existing repos. Do I need to modify reposilite.yml and rebuild docker image?

**Comment by dzikoysk** [comment]
Yeah, fair point, at this moment it is possible only if you modify `reposilite.yml`. I'll support lists in #112, thanks.

The documentation will be available on a Reposilite website (#79). I'm a little busy this week and I'm gradually implementing the dashboard (#17), but the website is a next priority.

**Comment by dzikoysk** [comment]
Ok, using the Reposilite 2.6.0 you should be able to specify list of proxied repositories by the following structure:

```bash
-Dreposilite.proxied="https://repo.panda-lang.org,https://jcenter.bintray.com"
```

Addresses are separated by comma.
